pkg/storaged/things-panel.jsx

```diff
@@ -55,19 +55,19 @@ export class ThingsPanel extends React.Component {
                 }
             }
 
-            return <StorageMenuItem onClick={install_then_action}>{title}</StorageMenuItem>;
+            return <StorageMenuItem key={title} onClick={install_then_action}>{title}</StorageMenuItem>;
         }
 
         const lvm2_feature = {
             is_enabled: () => client.features.lvm2
         };
 
         const actions = (
-            <StorageBarMenu id="devices-menu" label={_("Create devices")}>
-                { menu_item(null, _("Create RAID device"), () => create_mdraid(client)) }
-                { menu_item(lvm2_feature, _("Create volume group"), () => create_vgroup(client)) }
-                { menu_item(vdo_feature(client), _("Create VDO device"), () => create_vdo(client)) }
-            </StorageBarMenu>);
+            <StorageBarMenu id="devices-menu" label={_("Create devices")} menuItems={[
+                menu_item(null, _("Create RAID device"), () => create_mdraid(client)),
+                menu_item(lvm2_feature, _("Create volume group"), () => create_vgroup(client)),
+                menu_item(vdo_feature(client), _("Create VDO device"), () => create_vdo(client))].filter(item => item)} />
```

Comment:
What does the `filter` do here?

Reply:
Some items here are sometimes null, this PF4 component get's really upset about null children.

Reply:
Ah, makes sense. I had confused `filter` with `map`... d'oh.  What about writing `filter(item => item !== null)`?

Reply:
Sure I will replace on next repush.